packages/runtime/src/get-mesh.ts

```diff
@@ -1,25 +1,32 @@
-import { GraphQLSchema, execute, DocumentNode, GraphQLError } from 'graphql';
+import { GraphQLSchema, execute, DocumentNode, GraphQLError, isObjectType } from 'graphql';
 import { mergeSchemasAsync } from '@graphql-toolkit/schema-merging';
-import {
-  GraphQLOperation,
-  ExecuteMeshFn,
-  GetMeshOptions,
-  RawSourceOutput,
-  Requester
-} from './types';
+import { GraphQLOperation, ExecuteMeshFn, GetMeshOptions, RawSourceOutput, Requester } from './types';
 import {
   extractSdkFromResolvers,
   applySchemaTransformations,
   applyOutputTransformations,
-  ensureDocumentNode
+  ensureDocumentNode,
 } from './utils';
 import { Hooks, KeyValueCache } from '@graphql-mesh/types';
-import { addResolversToSchema } from 'graphql-tools-fork';
+import { addResolveFunctionsToSchema, IAddResolversToSchemaOptions } from 'graphql-tools-fork';
+
 import { InMemoryLRUCache } from '@graphql-mesh/cache-inmemory-lru';
-import {
-  applyResolversHooksToSchema,
-  applyResolversHooksToResolvers
-} from './resolvers-hooks';
+import { applyResolversHooksToSchema, applyResolversHooksToResolvers } from './resolvers-hooks';
+
+function addResolversWithReferenceResolver(options: IAddResolversToSchemaOptions) {
```

Comment:
move to a different file, and make sure to export this function :)

Reply:
Done!